packages/iTwinUI-react/src/core/Select/SelectTagContainer.tsx

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme, CommonProps, useOverflow, useMergedRefs } from '../utils';
+import SelectTag from './SelectTag';
+
+export type SelectTagContainerProps = {
+  /**
+   * Select tags.
+   */
+  tags: React.ReactNode[];
+} & Omit<CommonProps, 'title'>;
```

Comment:
This should use `Omit<React.ComponentPropsWithoutRef<'div'>, 'children'>`

Reply:
Changed